modules/couchdb/collect.go

```diff
@@ -112,6 +120,25 @@ func (cdb *CouchDB) scrapeActiveTasks(ms *cdbMetrics) {
 	ms.ActiveTasks = stats
 }
 
+func aggregateHTTPStatusCodes(metric string, collected map[string]int64, value int64) {
```

Comment:
i suggest to make `collected` first argument, if you write to something _dst_ is usually the first argument.

Reply:
Fixed